src/components/details.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+import Navbar from './Navbar';
+import '../styles/Details.css';
+
+const MyDetails = () => {
+  const [company, setCompany] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const params = useParams();
+  const { companyName } = params;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const response = await axios.get(`https://588fc30f7458d612002df0d2.mockapi.io/api/v1/companies?filter=${companyName}`);
+        setCompany(response.data);
+      } catch (error) {
+        console.error(error);
```

Comment:
[OPTIONAL]
- [ ] You may also consider adhering to the code best practice by deleting all the `console.log` in your codebase.